### PR TITLE
Armstrong number

### PR DESCRIPTION
**Pull request by satyampgt4**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description

this program will check whether a number is ARMSTRONG NUMBER or not

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#2036

### CODE REVIEW

File: JavaScript/Algorithms/Armstrong.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+process.stdin.resume();
```
Comment (HarshCasper) [inline JavaScript/Algorithms/Armstrong.js]: Add a beginner pitch about the Code with a brief background of the Algorithm

File: JavaScript/Algorithms/Armstrong.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    answer = answer + Math.pow(rem, n);
+    temp = Math.floor(temp / 10);
+  }
+
+  if (answer == number)
+    console.log(number + " is an Armstrong number.");
+  else
+    console.log(number + " is not an Armstrong number.");
+
+});
```
Comment (HarshCasper) [inline JavaScript/Algorithms/Armstrong.js:38]: Add sample I/O

File: JavaScript/README.md [hunk 3]
```diff
@@ -14,6 +14,7 @@
 - [Russian Peasant Algorithm](Algorithms/RussianPeasant.js)
 - [Z Algorithm](Algorithms/ZAlgorithm.js)
 - [Boyer Moore's Majority Vote Algorithm](Algorithms/BoyerMooreAlgorithm.js)
+- [Check Armstrong](Algorithms/Armstrong.js)
```
Comment (HarshCasper) [inline JavaScript/README.md]: Use `Armstrong Number` instead of `Check Armstrong`

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by satyampgt4** [comment]
@HarshCasper  requested changes are done please check

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**